python/paddle_fl/mpc/examples/cuda_demo/cuda_demo.py

```diff
@@ -0,0 +1,248 @@
+# Copyright (c) 2021 PaddlePaddle Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+CUDA ImageNet Demo
+"""
+
+import sys
+import numpy as np
+import time
+import logging
+
+import paddle
+import paddle.fluid as fluid
+import paddle.fluid.profiler as profiler
+import paddle_fl.mpc as pfl_mpc
+from paddle_fl.mpc.data_utils.data_utils import get_datautils
+
+import resnet
```

Comment:
seems not used

Reply:
uncomment line 117 as an option. any advice?